### PR TITLE
feat: online check

### PR DESCRIPTION
**Pull request by herrmannplatz**

### CODE REVIEW

File: electron/renderer/src/components/IsOnline.css [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    font-size: 16px;
+    font-weight: 300;
+    text-transform: uppercase;
+    color: #fff;
+ }
```
Comment (gregor) [inline electron/renderer/src/components/IsOnline.css]: EOF missing

File: electron/renderer/src/components/IsOnline.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+        }, { once: true });
+      }
+  }
+
+  render() {
+    return this.state.isOnline ? this.props.children : <div className="IsOnline">No Internet</div>;
+  }
+}
+
+export default IsOnline
```
Comment (gregor) [inline electron/renderer/src/components/IsOnline.js]: EOF missing

File: electron/renderer/src/components/IsOnline.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+    super(props)
+
+    this.state = { 
+        isOnline: navigator.onLine 
+    }
+  }
+
+  componentDidMount() {
+    if (this.state.isOnline === false) {
+        window.addEventListener('online', (event) => {
```
Comment (Yserz) [inline electron/renderer/src/components/IsOnline.js]: indentation

File: electron/renderer/src/components/IsOnline.js [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+import React, { Component } from 'react'
+
+import './IsOnline.css'
+
+class IsOnline extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = { 
+        isOnline: navigator.onLine
```
Comment (Yserz) [inline electron/renderer/src/components/IsOnline.js]: indentation

**Pull request merged**